docs/developer/payments.mdx

```diff
@@ -629,7 +629,7 @@ mutation TransactionEventReport {
 
 In the response, Saleor returns:
 
-- `alreadyProcessesed` - Defines if the reported event hasn't been processed earlier.
+- `alreadyProcessed` - Defines if the reported event hasn't been processed earlier.
```

Comment:
Could you also update this section to use colons instead of hyphens to separate definitions from field names, like the rest of the docs do?

Reply:
We can handle this in a separate PR because I see we're using a mix of `-` and `-` in multiple places.

Reply:
Moved that to https://github.com/saleor/saleor-docs/issues/913